connectors/di.js

```diff
@@ -27,6 +27,4 @@ Connector.getTrackArt = () => {
 	return null;
 };
 
-Connector.isPlaying = () => {
-	return $('#webplayer-region').attr('data-state') === 'playing';
-};
+Connector.playButtonSelector = '.icon-play';
```

Comment:
Could you revert this change? Now the extension recognizes now playing track as unknown one while loading.